### PR TITLE
DOC Remove support for MySQL 5

### PR DESCRIPTION
**Pull request by emteknetnz**
Issue https://github.com/silverstripe/silverstripe-framework/issues/11292

### CODE REVIEW

File: en/00_Getting_Started/00_Server_Requirements.md [hunk 1]
```diff
@@ -37,20 +37,11 @@ We officially support and regression test against the latest LTS releases of MyS
...
+### Connection mode (sql_mode) when using MySQL server >=8.0.0
 
-You can rectify this issue by upgrading MySQL, enabling the `innodb_large_prefix` setting if available, or reducing the
-size of affected fields. If none of these solutions are currently suitable, you can remove the collation
-configuration from `app/_config/mysite.yml` to default back to the legacy default collation.
-
-### Connection mode (sql_mode) when using MySQL server >=5.7.5
-
-In MySQL versions >=5.7.5, the `ANSI` sql_mode setting behaves differently and includes the `ONLY_FULL_GROUP_BY`
+In MySQL versions >=8.0.0, the `ANSI` sql_mode setting behaves differently and includes the `ONLY_FULL_GROUP_BY`
```
Comment (ChloeSartorelli) [inline en/00_Getting_Started/00_Server_Requirements.md]: Do we know what this means by "behaves differently"? I assume it means "behaves differently to older versions". If that's the case, we should remove that part so it just reads
> In MySQL versions >=8.0.0, the `ANSI` sql_mode setting  includes the ...

File: en/08_Changelogs/6.0.0.md [hunk 2]
```diff
@@ -339,6 +341,12 @@ Injector::inst()->load([
 - [`FieldList`](api:SilverStripe\Forms\FieldList) is now strongly typed. Methods that previously allowed any iterable variables to be passed, namely [`FieldList::addFieldsToTab()`](api:SilverStripe\Forms\FieldList::addFieldsToTab()) and [`FieldList::removeFieldsFromTab()`](api:SilverStripe\Forms\FieldList::removeFieldsFromTab()), now require an array to be passed instead.
 - [`BaseElement::getDescription()`](api:DNADesign\Elemental\Models\BaseElement::getDescription()) has been removed. If you had implemented this method in your custom elemental blocks, either set the [`description`](api:DNADesign\Elemental\Models\BaseElement->description) configuration property or override the [`getTypeNice()`](api:DNADesign\Elemental\Models\BaseElement::getTypeNice()) method.
 
+## Other changes
```
Comment (ChloeSartorelli) [inline en/08_Changelogs/6.0.0.md:344]: If we're having an "other changes" section, we should turn "Other new features and small changes" back into just "Other new features" and move the bit about `DBDecimal` to this new section.

File: en/08_Changelogs/6.0.0.md [hunk 3]
```diff
@@ -339,6 +341,12 @@ Injector::inst()->load([
...
+
+### MySQL 5 no longer supported {#mysql-5-support}
+
+MySQL 5.6 and 5.7  are no longer supported. The minimum supported version is MySQL 8.0. We support and test against against the latest LTS releases of MySQL and MariaDB.
```
Comment (ChloeSartorelli) [inline en/08_Changelogs/6.0.0.md]:
```suggestion
MySQL 5.6 and 5.7 are no longer supported. The minimum supported version is MySQL 8.0. We support and test against against the latest LTS releases of MySQL and MariaDB.
```

**Review by GuySartorelli — CHANGES_REQUESTED** [review]

File: en/00_Getting_Started/00_Server_Requirements.md (hunk 1 above)
Comment (emteknetnz) [reply en/00_Getting_Started/00_Server_Requirements.md]: Updated

File: en/08_Changelogs/6.0.0.md (hunk 2 above)
Comment (emteknetnz) [reply en/08_Changelogs/6.0.0.md:344]: Done

File: en/08_Changelogs/6.0.0.md (hunk 3 above)
Comment (emteknetnz) [reply en/08_Changelogs/6.0.0.md]: Fixed

File: en/08_Changelogs/6.0.0.md [hunk 4]
```diff
@@ -12,14 +12,17 @@ title: 6.0.0 (unreleased)
...
 - [Dependency changes](#dependency-changes)
   - [`intervention/image` has been upgraded from v2 to v3](#intervention-image)
 - [Bug fixes](#bug-fixes)
 - [API changes](#api-changes)
   - [Most extension hook methods are now protected](#hooks-protected)
   - [Strict typing for `Factory` implementations](#factory-strict-typing)
   - [General changes](#api-general)
+- [Other changes](#other-changes)
+  - [MySQL 5 no longer supported](#mysql-5-support)
+  - [DBDecimal default value](#dbdecimal-default-value)
```
Comment (ChloeSartorelli) [inline en/08_Changelogs/6.0.0.md]:
```suggestion
  - [`DBDecimal` default value](#dbdecimal-default-value)
```

File: en/08_Changelogs/6.0.0.md [hunk 5]
```diff
@@ -339,6 +341,16 @@ Injector::inst()->load([
 - [`FieldList`](api:SilverStripe\Forms\FieldList) is now strongly typed. Methods that previously allowed any iterable variables to be passed, namely [`FieldList::addFieldsToTab()`](api:SilverStripe\Forms\FieldList::addFieldsToTab()) and [`FieldList::removeFieldsFromTab()`](api:SilverStripe\Forms\FieldList::removeFieldsFromTab()), now require an array to be passed instead.
 - [`BaseElement::getDescription()`](api:DNADesign\Elemental\Models\BaseElement::getDescription()) has been removed. If you had implemented this method in your custom elemental blocks, either set the [`description`](api:DNADesign\Elemental\Models\BaseElement->description) configuration property or override the [`getTypeNice()`](api:DNADesign\Elemental\Models\BaseElement::getTypeNice()) method.
 
+## Other changes
+
+### MySQL 5 no longer supported {#mysql-5-support}
+
+MySQL 5.6 and 5.7 are no longer supported. The minimum supported version is MySQL 8.0. We support and test against against the latest LTS releases of MySQL and MariaDB.
+
+### DBDecimal default value {#dbdecimal-default-value}
```
Comment (ChloeSartorelli) [inline en/08_Changelogs/6.0.0.md]:
```suggestion
### `DBDecimal` default value {#dbdecimal-default-value}
```

**Review by GuySartorelli — CHANGES_REQUESTED** [review]

File: en/08_Changelogs/6.0.0.md (hunk 4 above)
Comment (emteknetnz) [reply en/08_Changelogs/6.0.0.md]: Updated

File: en/08_Changelogs/6.0.0.md (hunk 5 above)
Comment (emteknetnz) [reply en/08_Changelogs/6.0.0.md]: Updated

**Review by GuySartorelli — APPROVED** [review]
LGTM

**Pull request merged**